nncf/experimental/torch/sparsity/movement/structured_mask_handler.py

```diff
@@ -0,0 +1,344 @@
+            kept_col_blocks = F.max_pool1d(pruneable_cols.unsqueeze(0), kernel_size=self.grid_size[1]).squeeze(0)
+            head_id_to_keep = kept_col_blocks.nonzero().view(-1).cpu().numpy().tolist()
+
+        # prune bias only if ctx.prune_by_row is True
+        pruned_bias_shape = bias_shape
+        if self.sparsifier_operand.prune_bias and self.prune_by_row:
+            pruned_bias_shape = (int(self.sparsifier_operand.bias_ctx.binary_mask.count_nonzero().item()),)
+
+        return StructuredMaskContextStatistics(
+            weight_shape=weight_shape,
+            pruned_weight_shape=tuple(pruned_weight_shape),
+            bias_shape=bias_shape,
+            pruned_bias_shape=pruned_bias_shape,
+            head_or_channel_id_to_keep=head_id_to_keep,
+            module_node_name=self.module_node_name
+        )
+
+
+class SparsifiedModuleInfoGroup:
+    def __init__(self, group_id: int,
+                 group_type: BuildingBlockType,
+                 sparse_module_info: List[SparseModuleInfo]) -> None:
+        self.group_id = group_id
+        self.group_type = group_type
+        self.sparse_module_info = sparse_module_info
+
+
+class StructuredMaskContextGroup:
+    def __init__(self, group_id: int,
+                 group_type: BuildingBlockType,
+                 structured_mask_context_list: List[StructuredMaskContext]) -> None:
+        self.group_id = group_id
+        self.group_type = group_type
+        self.structured_mask_context_list = structured_mask_context_list
+
+    def __repr__(self) -> str:
+        str_ = f'[{self.group_id}]{self.group_type}: ['
+        for ctx in self.structured_mask_context_list:
+            str_ += f'\n\t{ctx}'
+        if len(self.structured_mask_context_list) == 0:
+            str_ += '<empty>'
+        str_ += '\n]'
+        return str_
+
+
+class StructuredMaskHandler:
+
+    def __init__(self,
+                 compressed_model: NNCFNetwork,
+                 sparsified_module_info_list: List[SparseModuleInfo],
+                 strategy: BaseStructuredMaskStrategy):
+        self.strategy = strategy
+        self.strategy_by_group_type = strategy.strategy_by_group_type
+        self.compressed_model = compressed_model
+        self.sparsified_module_info_list = sparsified_module_info_list
+
+        self._sparsified_module_info_groups = self._get_prunable_sparsified_module_info_group(compressed_model,
+                                                                                              sparsified_module_info_list)
+        self._structured_mask_ctx_groups = self._create_structured_mask_context_groups(
+            self._sparsified_module_info_groups,
+            self.strategy_by_group_type)
+
+        logger.debug('Structured mask contexts by group:')
+        for group in self._structured_mask_ctx_groups:
+            logger.debug(str(group))
+
+    @staticmethod
+    def _get_prunable_sparsified_module_info_group(
```

Comment:
We have an unspoken agreement for ordering method in classes similar to that:
https://stackoverflow.com/questions/10289461/what-is-a-good-way-to-order-methods-in-a-python-class#:~:text=Class%20methods%20and%20static%20methods,the%20class%20and%20come%20last.
It's easy to read the code when you know that public/interface methods go before some internal/utility ones.
Could you please reorder methods in all introduced classes following this agreement?